nodeup/pkg/model/ntp.go

```diff
@@ -57,103 +46,51 @@ func (b *NTPBuilder) Build(c *fi.ModelBuilderContext) error {
 		return nil
 	}
 
-	var ntpIP string
+	var ntpHost string
 	switch b.Cluster.Spec.CloudProvider {
 	case "aws":
-		ntpIP = "169.254.169.123"
+		ntpHost = "169.254.169.123"
 	case "gce":
-		ntpIP = "time.google.com"
+		ntpHost = "time.google.com"
 	default:
-		ntpIP = ""
+		ntpHost = ""
 	}
 
 	if b.Distribution.IsDebianFamily() {
-		c.AddTask(&nodetasks.Package{Name: "ntp"})
-
-		if ntpIP != "" {
-			bytes, err := updateNtpIP(ntpIP, ntpd)
-			if err != nil {
-				return err
-			}
-			c.AddTask(&nodetasks.File{
-				Path:     "/etc/ntp.conf",
-				Contents: fi.NewBytesResource(bytes),
-				Type:     nodetasks.FileType_File,
-				Mode:     s("0644"),
-			})
+		c.AddTask(&nodetasks.Package{Name: "chrony"})
```

Comment:
This will affect all debian and ubuntu distros, not just the most recent versions, correct? Are we sure the chrony package is available in older apt/yum repos?

I see it in [ubuntu](https://launchpad.net/ubuntu/+source/chrony), [debian](https://packages.debian.org/search?keywords=chrony), and [centos](https://pkgs.org/search/?q=chrony) so maybe we're confident enough that the package will be available on any supported distros. Thoughts?

Reply:
Chrony is available on all supported distros. It is installed by default on RHEL, CentOS and Debian. Ubuntu uses `systemd-timesyncd` by default though. To avoid having both running I updated the PR to configure it instead of Chrony for Ubuntu.

I tested this with the following distros and works:
* Amazon Linux 2
* CentOS Linux 7 (Core)
* Red Hat Enterprise Linux Server 7.4 (Maipo)
* Red Hat Enterprise Linux 8.1 (Ootpa)
* Debian GNU/Linux 9 (stretch)
* Debian GNU/Linux 10 (buster)
* Ubuntu 16.04.6 LTS
* Ubuntu 18.04.4 LTS
* Ubuntu Focal Fossa (development branch)

Reply:
We can also keep Chrony for Ubuntu, but timesyncd should be stopped and disabled.

Reply:
Ok, I think that is sufficient coverage 👍🏻